elytron/pom.xml

```diff
@@ -163,6 +163,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>commons-io</groupId>
```

Comment:
you mention `xom.io7m` but exclude `commons-io`. what's the relation between the 2?

Reply:
There is a compilation error ...

org.apache.commons:commons-collections4:jar:4.2:test
  \- commons-io:commons-io:jar:2.5:test

 does not exist in 5.9.0

Reply:
Thanks but where does `xom.io7m` come into the picture?

Reply:
For example here : https://github.com/wildfly/wildfly-core/blob/master/testsuite/shared/pom.xml#L176